src/views/WelcomeView.vue

```diff
@@ -23,93 +15,76 @@ import NcLoadingIcon from '@nextcloud/vue/dist/Components/NcLoadingIcon.js'
 import EmptyView from '../components/EmptyView.vue'
 
 import IconTalk from '../../img/app-dark.svg?raw'
+import { useStore } from '../composables/useStore.js'
+import Router from '../router/router.ts'
```

Comment:
Instead of direct import, `useRouter` should be user in Composition API, same as `this.$router` in Options API

Reply:
We are using Vue Router 3 still, and `useRouter` is only available from Vue Router 4.

Reply:
```suggestion
import { useRouter } from 'vue-router/composables'
```

- Source: https://github.com/vuejs/vue-router/blob/v3.6.5/src/composables/globals.js
- Export: https://github.com/vuejs/vue-router/blob/v3.6.5/package.json#L37